kong/clustering/services/debug.lua

```diff
@@ -0,0 +1,69 @@
+local _M = {}
+
+local get_log_level                = require("resty.kong.log").get_log_level
+local set_log_level                = require("resty.kong.log").set_log_level
+local constants                    = require("kong.constants")
+
+local LOG_LEVELS                   = constants.LOG_LEVELS
+local DYN_LOG_LEVEL_KEY            = constants.DYN_LOG_LEVEL_KEY
+local DYN_LOG_LEVEL_TIMEOUT_AT_KEY = constants.DYN_LOG_LEVEL_TIMEOUT_AT_KEY
```

Comment:
Let's not align ` = ` with new code files.
```suggestion
local get_log_level = require("resty.kong.log").get_log_level
local set_log_level = require("resty.kong.log").set_log_level
local constants = require("kong.constants")

local LOG_LEVELS = constants.LOG_LEVELS
local DYN_LOG_LEVEL_KEY = constants.DYN_LOG_LEVEL_KEY
local DYN_LOG_LEVEL_TIMEOUT_AT_KEY = constants.DYN_LOG_LEVEL_TIMEOUT_AT_KEY
```

Reply:
Addressed.